src/app/learn/get-started/basics/serve-project/index.html

```diff
@@ -54,5 +54,7 @@
       </p>
     </li>
   </ol>
-
+  <p>
+    To enable Prod mode run <stache-code>skyux build --serve</stache-code> (remove dist directory if it already exist).
```

Comment:
Does "Prod" need to be capitalized and abbreviated? I think we should change this to "production." ... There should be a comma after "mode." ... I'd put a period after the closing stache-code tag, capitalize "Remove," and move the period at the end inside the parentheses. ... Add "the" before "dist" and wrap "dist" in stache-code tags. ... Change "exist" to "exists." ... And I'd add another sentence between the two current sentences: "This builds the current SPA into the dist/ folder and then serves it."

Here's a version with all my suggested changes in place: 
"To enable production mode, run <stache-code>skyux build --serve</stache-code>. This builds the current SPA into the dist/ folder and then serves it. (Remove the <stache-code>dist<stache-code> directory if it already exists.)"